.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: required
 
 language: scala
 scala:
-   - 2.11.6
+   - 2.11.8
```

Comment:
I believe we should be using Scala 2.11.11. See here https://github.com/apache/incubator-openwhisk/blob/master/settings.gradle#L23.